ui/src/test/org/jboss/windup/web/selenium/Selenium06Test.java

```diff
@@ -0,0 +1,181 @@
+		String a = "/home/mbrophy/Sample_Files/06__all_apps/01/AdditionWithSecurity-EAR-0.01.ear";
+		selenium.robotSelectFile(a);
+		// checks that the uploaded file is green and has the correct information.
+		assertEquals("AdditionWithSecurity-EAR-0.01.ear (36.11 MB):rgba(63, 156, 53, 1)", selenium.checkFileInfo(2));
+
+		String b = "/home/mbrophy/Sample_Files/06__all_apps/01/arit-ear-0.8.1-SNAPSHOT.ear";
+		selenium.robotSelectFile(b);
+		assertEquals("arit-ear-0.8.1-SNAPSHOT.ear (3.978 MB):rgba(63, 156, 53, 1)", selenium.checkFileInfo(3));
+
+		selenium.robotCancel();
+		assertTrue(selenium.nextEnabled());
+
+		/*
+		 * Step 05
+		 */
+		assertTrue(selenium.nextEnabled());
+		selenium.clickNext();
+
+		assertEquals("Migration to JBoss EAP 7", selenium.transformationPath());
+
+		//looks through the first level of the include packages 
+		assertEquals(
+				"1\nantlr\ncom\njavassist\njavax\njunit\nmx\nnet\noracle\norg\nrepackage\nschemaorg_apache_xmlbeans",
+				selenium.findPackages());
+		// checks that the three more detailed dialogue are compressed
+		assertTrue(selenium.collapesdInfo());
+
+		//this will go through the packages, to the bottom level and check then uncheck it.
+		assertTrue(selenium.testPackages(1));
+		assertFalse(selenium.testEmptyPackages(1));
+
+		//opens the exclude packages section
+		assertTrue(selenium.isCollapsed("Exclude packages"));
+		selenium.clickCollapsed("Exclude packages");
+		assertFalse(selenium.isCollapsed("Exclude packages"));
+
+		//this will go through the package system under exclude packages
+		assertTrue(selenium.testPackages(2));
+		assertFalse(selenium.testEmptyPackages(2));
+
+		//opens the Advanced options section
+		selenium.clickCollapsed("Advanced options");
+
+		//uner advanced options, this adds a new option
+		selenium.addOptions();
+		selenium.optionsDropdown("enableCompatibleFilesReport");
+		selenium.toggleValue(1);
+		selenium.addOption(1);
+		assertTrue(selenium.value(1));
+
+		//runs the project with the above specifications
+		selenium.saveAndRun();
+		assertTrue(selenium.checkProgressBar());
+		assertTrue(selenium.analysisResultsComplete(1));
+		
+		selenium.closeDriver();
+
+	}
+
+	public void testReports() throws InterruptedException, AWTException {
+		AppLevel selenium = new AppLevel();
+
+		selenium.navigateProject("test 3");
+		Thread.sleep(5000);
+		selenium.clickAnalysisReport(1);
+		selenium.navigateTo(1);
+		selenium.clickApplication("AdministracionEfectivo.ear");
+
+		ArrayList<String> list = new ArrayList<String>();
+		list.add("All Applications");
+		list.add("Dashboard");
+		list.add("Issues");
+		list.add("Application Details");
+		list.add("Technologies");
+		list.add("Unparsable");
+		list.add("Dependencies");
+		list.add("Compatible Files");
+		list.add("EJBs");
+		list.add("JPA");
+		list.add("Server Resources");
+		list.add("Hard-coded IP Addresses");
+		list.add("Ignored Files");
+		list.add("About");
+		list.add("Send Feedback");
```

Comment:
Please add `list.add("Tattletale");` because it is generated by default when EAP is one of the targets